nodejs/mongodb-driver/mongodb.js

```diff
@@ -11,16 +11,57 @@ const client = new MongoClient(uri, {
 client.connect((err) => {
   const statistics = client.db("covid19").collection("statistics");
 
-  // find the latest 15 cases from France
+  // // Query to get the last 5 entries for France (continent only)
   statistics
     .find({ country: "France" })
-    .sort([["a", 1]])
+    .sort([["a", -1]])
     .limit(15)
     .toArray(function (err, docs) {
       if (err) {
         console.error(err);
       }
       console.log(docs);
-      client.close();
+    });
+
+  //Query to get the last day data (limited to 15 docs here).
+  statistics
+    .find({
+      date: {
+        $lt: new Date(),
+        $gt: new Date(new Date().setDate(new Date().getDate() - 2)),
```

Comment:
Will not work if we don't update every day or JHU miss an update. That's why I get the "lastdate" from the metadata in Java code. But detail. Just FYI